.circleci/config.yml

```diff
@@ -2129,48 +2125,26 @@ jobs:
           command: |
             $ProgressPreference = "SilentlyContinue"
             mkdir python_installer -Force
-            if (!(Test-Path .\python_installer\python-2.7.18.amd64.msi -PathType Leaf)) {
-                wget -O .\python_installer\python-2.7.18.amd64.msi https://www.python.org/ftp/python/2.7.18/python-2.7.18.amd64.msi
+            if (!(Test-Path .\python_installer\python-3.8.7-amd64.exe -PathType Leaf)) {
+                wget -O .\python_installer\python-3.8.7-amd64.exe https://www.python.org/ftp/python/3.8.7/python-3.8.7-amd64.exe
```

Comment:
Just curious how we have decided which Python 3 version to use?

Since we have freedom which one to use, perhaps we could just use the latest one (3.9.x)? Or some modules we depend on are not compatible with it yet?